modules/configuration-api/src/main/java/org/apache/ignite/configuration/ConfigurationModule.java

```diff
@@ -78,12 +78,12 @@ public interface ConfigurationModule {
     }
 
     /**
-     * Returns classes of internal schema extensions (annotated with {@link InternalConfiguration})
-     * provided by this module.
+     * Returns classes of schema extensions (annotated with {@link ConfigurationExtension})
+     * provided by this module, including internal extensions.
      *
-     * @return internal schema extensions' classes
+     * @return all schema extensions' classes including internal
      */
-    default Collection<Class<?>> internalSchemaExtensions() {
+    default Collection<Class<?>> allSchemaExtensions() {
```

Comment:
Polymorphic configuration is also an extension of schemas, the name is not quite suitable, I think we need to remove the `all` prefix.